client/src/components/graph/overlays/graphOverlayLayer.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component, cloneElement } from "react";
+import { connect } from "react-redux";
+import { mat3 } from "gl-matrix";
+
+import styles from "../graph.css";
+
+export default
+@connect(state => ({
+  responsive: state.responsive
+}))
+class GraphOverlayLayer extends Component {
```

Comment:
Any reason this is not a PureComponent?

Reply:
none that I can see, changed to utilize PureComponent